.github/actions/deploy-happy-stack/action.yml

```diff
@@ -1,37 +1,38 @@
 name: Create or Update a Happy Stack
 description: "Will create or update a Happy Stack"
 inputs:
-  tfe-token:
-    description: "TFE Token"
-    required: true
   stack-name:
     description: "Name of the stack to create or update"
     required: true
   create-tag:
     description: "Should we build and tag docker images. Defaults to false and assumes images already exist."
     default: "false"
     required: false
+  tfe-token:
+    description: "TFE Token"
+    required: true
 runs:
   using: "composite"
   steps:
     - uses: actions/checkout@v2
     - name: Install happy
-      uses: chanzuckerberg/github-actions/.github/actions/install-happy@41c7efb56fadc2517f6843a50c1d42bd38a47b9a
+      uses: chanzuckerberg/github-actions/.github/actions/install-happy@9f6d901750728bf97041063580eeda064ff74322
```

Comment:
How do we pull the right version of happy cli?

Reply:
there's an argument you can specify, currently defaults to 0.15.3. 
I think I want to make it default to "latest" and instead have users pin a version if they want